main.js

```diff
@@ -0,0 +1,32 @@
+const menuIconMobile = document.querySelector('.navbar__hamb');
+const menu = document.querySelector('.navbar__menu');
+const viewport = document.querySelector('body');
+const menuLinks = document.querySelectorAll('.navbar__menu__link');
+
+function isActiveClassExisted(element) {
+  return element.classList.contains('active');
+}
+
+function refreshIcon() {
+  if (isActiveClassExisted(menu)) {
+    menuIconMobile.setAttribute('src', 'images/hamburguer-close.png');
+  } else {
+    menuIconMobile.setAttribute('src', 'images/hamburguer.png');
+  }
+}
+function openMenu() {
+  menu.classList.toggle('active');
+  viewport.classList.add('blockover');
+  refreshIcon();
+}
+
+function closeMenu() {
```

Comment:
[OPTIONAL]

- I do recommend using ES6 syntax for javascript, but it's okay if you haven't learned it yet. It will make your code cleaner. 👍